packages/proto-loader/bin/proto-loader-gen-types.ts

```diff
@@ -598,7 +598,7 @@ function generateLoadedDefinitionTypes(formatter: TextFormatter, namespace: Prot
 
 function generateRootFile(formatter: TextFormatter, root: Protobuf.Root, options: GeneratorOptions) {
   formatter.writeLine(`import * as grpc from '${options.grpcLib}';`);

```

Comment:
```suggestion
  formatter.writeLine(`import type * as grpc from '${options.grpcLib}';`);

```

Reply:
Done, I also addressed the other instance of the same import line and fixed another bug. That's all now available in 0.6.0-pre17.